labs/fabric-debugger.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: VS Code Plugin for Chaincode Developers
+parent: Labs
+---
+# Lab Name
+VS Code Plugin for Chaincode Developers
+
+
+# Short Description
+A VS code extension and toolbox to debugging a chaincode. The extension once installed can be used by developers of chaincode to run and debug from their IDE, to understand how their chaincode behaves.
+
+# Scope of Lab
+Hyperledger Fabric chaincode developers often have to repeat the same steps to set up their environment and deploy code for debugging. This process can be tedious and time-consuming. If the IDE had a built-in debugging interface, it could automate these tasks, making the developer's job easier and faster.

```

Comment:
This lab sounds like a good lab. It would be nice to detail the full scope of this lab here. Is this just for debugging or will it also attempt to create a Fabric deployment.?

Reply:
Sure @tkuhrt , I will work with @urgetolearn to add more details to the scope. Deployment of a Fabric network is not in the scope of this lab. The goal is to enable debugging capabilities on the VS Code IDE, following the adaptable protocol so that the extension can be reused across other IDEs. There will be a shortcut to bring up a fabric-samples-based network, but the debugger is designed to work with any Fabric network, deployed with any tool of the developer's choice.

Reply:
Great. I will wait to see those updates before approving this lab. Looking forward to seeing what is accomplished.